packages/fhir-keycloak-user-management/src/components/UserList/TableActions/utils.tsx

```diff
@@ -3,33 +3,197 @@ import { sendErrorNotification, sendSuccessNotification } from '@opensrp/notific
 import { removeKeycloakUsers } from '../../../ducks/user';
 import { KEYCLOAK_URL_USERS } from '../../../constants';
 import lang, { Lang } from '../../../lang';
+import { FHIRServiceClass } from '@opensrp/react-utils';
+import { IPractitioner } from '@smile-cdr/fhirts/dist/FHIR-R4/interfaces/IPractitioner';
+import { IPractitionerRole } from '@smile-cdr/fhirts/dist/FHIR-R4/interfaces/IPractitionerRole';
+import { ICareTeam } from '@smile-cdr/fhirts/dist/FHIR-R4/interfaces/ICareTeam';
+import { CareTeamParticipant } from '@smile-cdr/fhirts/dist/FHIR-R4/classes/careTeamParticipant';
 
 /**
- * Delete keycloak user
+ * Delete keycloak user and practitioner
  *
  * @param {Function} removeKeycloakUsersCreator - remove users action creator
  * @param {string} keycloakBaseURL - keycloak api base URL
+ * @param fhirBaseURL - fhir api base url
  * @param {string} userId - id of user to be deleted
  * @param {Function} isLoadingCallback - callback function that sets loading state
  * @param {Lang} langObj - lang
  * @returns {void}
  */
-export const deleteUser = (
+export const deleteUser = async (
   removeKeycloakUsersCreator: typeof removeKeycloakUsers,
   keycloakBaseURL: string,
+  fhirBaseURL: string,
   userId: string,
   isLoadingCallback: (loading: boolean) => void,
   langObj: Lang = lang
-): void => {
-  const serviceDelete = new KeycloakService(`${KEYCLOAK_URL_USERS}/${userId}`, keycloakBaseURL);
-  serviceDelete
-    .delete()
+) => {
+  // start loader
+  isLoadingCallback(true);
```

Comment:
this should be in the view component

Reply:
@ashfahan 09cdac75eb2acbdf5b7e6f7f0d68ab690a5cbb4b